types/make-fetch-happen/make-fetch-happen-tests.ts

```diff
@@ -0,0 +1,74 @@
+import fetcher from 'make-fetch-happen';
```

Comment:
When you turn `esModuleInterop` off this should become

```suggestion
import fetcher = require('make-fetch-happen');
```